documentation/assemblies/upgrading/assembly-upgrade.adoc

```diff
@@ -16,7 +16,7 @@ Strimzi can be xref:assembly-downgrade-{context}[downgraded] to the previous ver
 
 Two upgrade paths are possible:
 
-Incremental:: 
+Incremental::
 Upgrading Strimzi from the previous minor version to version {ProductVersion}.
 
 Multi-version::
```

Comment:
@scholzj Does multi-version change? Do we need to go to 0.23, update our resources and then to 0.24?

Reply:
There are two options:
1) Upgrade to 0.22, covert the CRDs and upgrade to 0.23 or any newer version
2) Install only the CRDs from 0.22, convert the CRDs, and upgrade to 0.23 or any newer version

It is probably fine to document only the first option. But both are valid in general.

Reply:
I've added a new sub-section, with the second alternative as a note.
I've also updated the first step in the _Required upgrade sequence_ to mention the order if upgrading prior to version 0.22